src/context/chatty/ChattyProvider.js

```diff
@@ -65,7 +65,7 @@ function ChattyProvider({children}) {
         // sort by activity, pinned first
         nextThreads = nextThreads
             .sort((a, b) => maxPostIdByThread[b.threadId] - maxPostIdByThread[a.threadId])
-            .sort((a, b) => a.markType === b.markType === 'pinned' ? 0 : a.markType === 'pinned' ? -1 : 1)
+            .sort((a, b) => a.markType === b.markType === 'pinned' || a.markType === b.markType !== 'pinned' ? 0 : a.markType === 'pinned' ? -1 : 1)
```

Comment:
Huh. I see what the issue is here, and it was probably a refactoring error on my part. This a = b = c thing isn't intended at all.

Reply:
I'll play with this today, and restore this line to something that isn't insane-JS.